lib/processes.js

```diff
@@ -97,6 +103,17 @@ function launchAllProcesses(config) {
         args.push('--headless');
       }
       if (process.getuid() === 0) args.push('--no-sandbox');
+
+      var debugPort = config.get('chrome.remoteDebuggingPort', null);
```

Comment:
Since this already runs only when we're using chrome - any reason not to default this to `true` instead of `null`?

Reply:
@jkrems This assumes that if the port has been set (with actual value) in _.testiumrc_, it would address that. Otherwise would discover the new port and update the config.

Are you suggesting that to remove the check and find new port anyway?

Reply:
Is there value in making the port configurable? That seems to encourage hard-coding values which might lead to problems when running multiple test suites etc..

Reply:
Oh yes. I didn't think about that. I'll remove the checks